packages/react/src/contexts/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { Children, createContext, Fragment, isValidElement, PropsWithChildren, ReactNode, useContext } from "react";
+import type { RoboWizard, StepConfig } from "robo-wizard";
+import { useWizard } from "../hooks";
+
+/**
+ * @param props - accepts same props to match [[StepConfig]] for [[useWizard]], as well as the `element` to display
+ **/
+export function Step(_props: StepConfig & { element: ReactNode | null }): React.ReactElement | null {
```

Comment:
This is "component" is just an object for holding props as configuration; similar to the Route component from React Router.